src/components/TheUserMenu.vue

```diff
@@ -16,6 +17,10 @@
           class="mr-1"
         />
         {{ authStore.username }}
+        <v-icon
+          :icon="isOpen ? 'mdi-chevron-up' : 'mdi-chevron-down'"
+          class="ml-1"
```

Comment:
This is exactly what was asked, but in looking at it, I think it would be slick to keep the icon the same all the time (mdi-chevron-up), and to apply a class to rotate it when `isOpen` is true. 

We could give this element a 
```
transition: transform .5s ease-in-out;
```

And then dynamically apply a class when isOpen is true to rotate it like

```
.rotate {
  transform: rotate(180deg);
}
```

That should give it a nice rotating animation when it transitions between states

Reply:
Thank you for your suggestion, the icon is transformed more smoothly now, but .5s is a bit slow so I put .3s instead. I'm just a newcomer so I'm very grateful to have a little contribution to your project! Love youuuu❤️